.github/workflows/test.yml

```diff
@@ -24,4 +25,20 @@ jobs:
         with:
           name: holochain-ci
           authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
-      - run: nix develop --command ${{ matrix.script }}
+      - run: nix develop --command ./${{ matrix.script }}-${{ matrix.wasmer-feature }}.sh
+
+  test-windows:
```

Comment:
- Test for both wasmer features on windows
- Move windows job into same file to make it more obvious that both jobs need to be modified together